tools/platyPS/GenerateHelp.ps1

```diff
@@ -0,0 +1,133 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True, Position=0)]
+    [String]$Service,
+    [Parameter(Mandatory=$True, Position=1)]
+    [String]$PathToRepo
+)
+
+# Check to see if platyPS is already an installed module
+if ((Get-Module -ListAvailable -Name platyPS).Count -eq 0)
+{
+    Write-Host "Installing platyPS"
+    Install-Module -Name platyPS -scope CurrentUser
+    Import-Module platyPS
+}
+
+# If we can't find a path for the given service, then it doesn't exist in AzureRM
+if (!(Test-Path $PathToRepo\src\ResourceManager\$Service\Commands.$Service))
+{
+    throw "Cannot find path to $PathToRepo\src\ResourceManager\$Service\Commands.$Service"
+}
+
+# Import service module
+Import-Module "$PathToRepo\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.$Service\AzureRM.$Service.psd1"
```

Comment:
We should make BuildTarget (Debug/Release) a parameter, with default value Debug

Also, we should be able to use this tool to update help for ServiceManagement and Storage cmdlets as well.  Perhaps we could use a separate parameter set, with just the path to the psd1 file?